src/lib/routes/admin-api/rbac.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const Controller = require('../controller');
```

Comment:
TODO: remove this in the initial release (not needed before we address custom roles)